articles/app-service/deploy-staging-slots.md

```diff
@@ -338,6 +338,8 @@ Get-AzLog -ResourceGroup [resource group name] -StartTime 2018-03-07 -Caller Slo
 Remove-AzResource -ResourceGroupName [resource group name] -ResourceType Microsoft.Web/sites/slots –Name [app name]/[slot name] -ApiVersion 2015-07-01
 ```
 
+To perform a slot swap from the production slot, the identity needs (at minimum) permissions to perform the `Microsoft.Web/sites/slotsswap/Action` operation. For more information, see the [Resource provider operations](../role-based-access-control/resource-provider-operations.md#microsoftweb)
```

Comment:
Might be nice to mention a "minimal" built-in role with that permission. It's not always an option to create a custom role.

I _guess_ that would be [Website contributor](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#website-contributor), but not sure (and might depend in individual point of view on minimality).